rdp-server/frontend.cpp

```diff
@@ -386,59 +391,63 @@ static BOOL ogon_peer_post_connect(freerdp_peer *client)
 		return FALSE;
 	}
 
-	if (client->settings->AutoLogonEnabled)	{
+	if (client->context->settings->AutoLogonEnabled) {
```

Comment:
here and below that could be simply `settings->` instead of `client->context->settings->`